web/ui/static/js/graph.js

```diff
@@ -1,16 +1,18 @@
 var Prometheus = Prometheus || {};
-var graphs = [];
 var graphTemplate;
 
 var SECOND = 1000;
 
 Handlebars.registerHelper('pathPrefix', function() { return PATH_PREFIX; });
 
-Prometheus.Graph = function(element, options) {
+Prometheus.Graph = function(element, options, changeHandler, removeHandler) {
```

Comment:
Actually in the caller you call them `updateURL()` and `removeGraph()`, which I think would be good names here too.